config.toml

```diff
@@ -48,11 +48,11 @@ windows_dl_link = "https://downloads.redisinsight.redislabs.com/latest/redisinsi
 docker_command = "docker run -v redisinsight:/db -p 8001:8001 redislabs/redisinsight:latest"
 docker_db_volume_permission = "chown -R 1001 redisinsight"
 
-rdi_redis_gears_min_semantic_version = "1.2.5"
-rdi_redis_gears_min_version = "10205"
+rdi_redis_gears_min_semantic_version = "1.2.6"
+rdi_redis_gears_min_version = "10206"
 rdi_redis_gears_current_semantic_version = "1.2.6"
 rdi_redis_gears_current_version = "10206"
```

Comment:
do we really need the 4 of them? Seems like we just need the `min_version` and `min_semantic_version` and update accordingly. No real need to say that we support 1.2.5 but the current one is 1.2.6, let's keep it simple.